tests/test-cases/Group1-Docker-Commands/1-04-Docker-Create.robot

```diff
@@ -19,6 +19,17 @@ Suite Setup  Conditional Install VIC Appliance To Test Server
 Suite Teardown  Cleanup VIC Appliance On Test Server
 Test Timeout  20 minutes
 
+*** Keywords ***
+# NOTE: this function should only be used in the case where you supply a name to the container. Since inventory names result in `<cvm-name>-<cvm-uuid>`
+Check CVM Inventory Path
+    [Arguments]  ${cvm-name}
+    ${rc}  ${cvm-path}=  Run And Return Rc And Output  govc find / -type m | grep ${cvm-name}
+    Should Be Equal As Integers  ${rc}  0
+    # If it is esxi - we should find the vch in the vmfolder
+    Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Should Contain  ${cvm-path}  vm/${cvm-name}
```

Comment:
Are these `HOST_TYPE` checks needed now that we're off vApps, or are they for backwards compatibility? Or something else? Just curious.

Reply:
Inventory folders are not supported on standalone esxi. So we make everything at the vmfolder elevel if we are targeting just esxi. Otherwise we expect a namespaced path.

Reply:
@cgtexmex I just realized I used the word inventory there. I'll change it when I get the chance.